samples/constitutions/default/actions.js

```diff
@@ -762,6 +762,25 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "set_js_engine_options",
+    new Action(
+      function (args) {
+        checkType(args.max_heap_bytes, "integer", "max_heap_bytes");
+        checkType(args.max_stack_bytes, "integer", "max_stack_bytes");
+      },
+      function (args) {
+        const js_engine_map = ccf.kv["public:ccf.gov.jsengine"];
```

Comment:
If this PR creates a new KV map (which I think it does?), it should be documented here: https://microsoft.github.io/CCF/main/audit/builtin_maps.html

Reply:
Done